core/domain/stats_jobs_continuous.py

```diff
@@ -250,6 +294,19 @@ def map(item):
 
     @staticmethod
     def reduce(key, stringified_values):
+        """Implements the reduce function. Must be declared @staticmethod.
+
+        Args:
+            key: str. A key value as emitted from the map() function, above. It
```

Comment:
str. The exploration id and version of the exploration in the form 'exploration_id.version'.